ord_schema/validations.py

```diff
@@ -562,14 +637,21 @@ def validate_reaction_outcome(message):
                         f'Undefined analysis key {key} '
                         'in ReactionProduct', ValidationError)
     if not message.products and not message.HasField('conversion'):
-        # TODO(kearnes): Should products and conversion be mutually exclusive?
         warnings.warn(
             'No products or conversion are specified for reaction;'
-            ' at least one must be specified', ValidationError)
+            ' this is permissible only for multistep reactions',
+            ValidationWarning)
 
 
 def validate_reaction_product(message):
-    del message  # Unused.
+    # pylint: disable=too-many-boolean-expressions
+    if (message.compound.HasField('volume_includes_solutes')
```

Comment:
Someone will need to go through and update the validation docs; not in this PR :)